install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -75,11 +75,15 @@ spec:
             value: "false"
           - name: "ENABLE_INGRESS_GATEWAY_SDS"
             value: "true"
-          - name: "INGRESS_GATEWAY_NAMESPACE"
+          - name: "INGRESS_GATEWAY_SECRETS_NAMESPACE"
+{{- if $spec.sds.secretsNamespace }}
```

Comment:
change to `if not`?

Reply:
Ah good catch. I flipped the contents to avoid the use of `if not`.